firestore/angular-rewrite/README.md

```diff
@@ -103,6 +103,14 @@ Follow these steps to have a working version of the quickstart running with the
  ng serve
  ```
 
+ 11. (Optional) Populate production Firestore with mock data
+
+ Now that the application is live on the Firebase console, you likely want to add some data to your production Firestore database so that you can see the application in action. Rather than manually adding restaurants and reviews, a convenice script is available to populate your production Firestore database. Run the following command (defined in the `Firestore/` directory's `package.json`) to add mock data:
```

Comment:
nit: let's note that this requires your configuration to exist in the config file